MatrixSDK/Utils/MXTools.m

```diff
@@ -80,6 +81,8 @@ + (void)initialize
                           kMXEventTypeStringReadMarker: @(MXEventTypeReadMarker)
                           };
 
+        revertEventTypesMap = eventTypesMap.allKeys;
```

Comment:
CAUTION: According to ios doc, the order of the elements in the array is not defined.